app/code/core/Mage/Adminhtml/Block/Widget/Grid/Column/Renderer/Abstract.php

```diff
@@ -123,16 +126,16 @@ public function renderHeader()
     {
         if ($this->getColumn()->getGrid()->getSortable() !== false && $this->getColumn()->getSortable() !== false) {
             $className = 'not-sort';
-            $dir = strtolower($this->getColumn()->getDir());
+            $dir = strtolower((string)$this->getColumn()->getDir());
             $nDir = ($dir == 'asc') ? 'desc' : 'asc';
             if ($this->getColumn()->getDir()) {
                 $className = 'sort-arrow-' . $dir;
             }
             $out = '<a href="#" name="' . $this->getColumn()->getId() . '" title="' . $nDir
                    . '" class="' . $className . '"><span class="sort-title">'
-                   . $this->escapeHtml($this->getColumn()->getHeader()) . '</span></a>';
+                   . $this->escapeHtml((string)$this->getColumn()->getHeader()) . '</span></a>';
```

Comment:
```suggestion
                   . $this->escapeHtml($this->getColumn()->getHeader()) . '</span></a>';
```
Revert after https://github.com/OpenMage/magento-lts/pull/2884